sdk/eventhub/event-hubs/package.json

```diff
@@ -63,7 +63,7 @@
   },
   "dependencies": {
     "@azure/abort-controller": "1.0.0-preview.1",
-    "@azure/core-amqp": "1.0.0-preview.2",
+    "@azure/core-amqp": "1.0.0-preview.3",
```

Comment:
Do we want to bump this dependency now?  Are we certain we will ship `core-amqp@preview.3` before `event-hubs@preview.3`?

Reply:
These 2 are developed together hand in hand.
So if any changes are going into core-amqp, it will be used in event hubs